app/javascript/components/donations/StripeForm.tsx

```diff
@@ -60,14 +60,17 @@ export function StripeForm({
   })
 
   const cardOptions = generateCardOptions(useStripeFormTheme())
+  const paymentElementOptions = {
+    layout: {
+      type: 'accordion',
+      defaultCollapsed: false,
+      radios: true,
+      spacedAccordionItems: false,
+    },
+  }
 
   return (
-    <form
-      data-turbo="false"
-      onSubmit={
-        paymentIntentType === 'payment' ? handlePaymentSubmit : handleSubmit
-      }
-    >
+    <form data-turbo="false" onSubmit={handlePaymentSubmit}>
```

Comment:
You can get rid of `handleSubmit` entirely now, and probably a whole ton of unused code